packages/core/admin/admin/src/components/FormInputs/types.ts

```diff
@@ -31,8 +38,12 @@ interface InputProps {
         | 'dynamiczone'
         | 'component'
         | 'relation'
+        | 'text'
+        | 'string'
+        | 'password'
+        | 'email'
```

Comment:
I think it's important to have it derived from `Attribute.Kind` because if we add a new attribute we'll catch not handling it etc. Would your allowlist be a unique list or derived from `Attribute.Kind`?